_i18n/ru.yml

```diff
@@ -7,19 +7,22 @@ base:
             title: "Часто задаваемые вопросы"
             text: "FAQ"
         takepart: "Примите&nbsp;участие"
+        language: "Язык"
 
 index:
     title: "Добро пожаловать"
-    
+    description: |
+        Bitcoin XT представляет собой реализацию полной ноды, которая соответствует первоначальному видению Биткоина: простые, надёжные, недорогие транзакции, доступные каждому в мире.
     header:
         scalingicon: "Масштабирование<br>Биткоина"
         usersicon: "Приоритет<br>за пользователями"
         communityicon: "Прагматичное<br>сообщество"
         info: |
-            Bitcoin XT - это воплощение полной ноды биткоин-сети, которая соответствует первоначальному видению Биткоина: простые, надёжные, дешёвые транзакции для каждого в мире. Изначально Bitcoin XT представлял из себя простой набор патчей поверх Bitcoin Core, сейчас же Bitcoin XT - независимый форк с множеством новых <a href="/ru/patches.html">возможностей</a>. Для того, чтобы каждый мог проверить, что загружаемые файлы согласуются с <a href="https://github.com/bitcoinxt/bitcoinxt">исходным кодом</a>, используется метод <a href="https://github.com/bitcoinxt/gitian.sigs">Deterministic Builds</a>.
+            Bitcoin XT - это воплощение полной ноды Bitcoin Cash (BCH) и Bitcoin (BTC). Bitcoin XP соответствует первоначальному видению Биткоина: простые, надёжные, дешёвые транзакции для каждого в мире. Изначально Bitcoin XT представлял собой простой набор патчей поверх Bitcoin Core, сейчас же Bitcoin XT - это независимо разрабатываемый форк. Для того, чтобы каждый мог проверить, что загружаемые файлы согласуются с <a href="https://github.com/bitcoinxt/bitcoinxt">исходным кодом</a>, используется метод <a href="https://github.com/bitcoinxt/gitian.sigs">Deterministic Builds</a>.
```

Comment:
Replace Bitcoin XP with Bitcoin XT

Reply:
Sure, sorry for this typo. I will fix it tomorrow. I want to review the translation again and change some words in this discription. 

Actually, I do not like this description in English. Too many "Bitcoin" words there and they stay too close together. Also I do not like calling "Bitcoin XT" as fork here... Maybe "project" will be better?